vcr/pe/schema/v2/presentation-definition-claim-format-designations.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Presentation Definition Claim Format Designations",
+  "type": "object",
+  "additionalProperties": false,
+  "patternProperties": {
+    "^jwt$|^jwt_vc$|^jwt_vp$": {
```

Comment:
Does OpenID4VP specify those for the exact same fields/schema?

If they clash, since OpenID4VP is a draft, I'd stick with the PEX specification since that's the source of the schemas (and thus for allowed values, I'd say).

Reply:
it does. Also see ongoing discussion in #2447.

I can live with PEX specified values only though